Dockerfile

```
@@ -0,0 +1,46 @@
+COPY apps/service-bun/package.json apps/service-bun/package.json
+COPY packages/core/package.json packages/core/package.json
+COPY packages/integrations/package.json packages/integrations/package.json
+COPY packages/rpc/package.json packages/rpc/package.json
+
+RUN pnpm install --frozen-lockfile
+
+COPY apps ./apps
+COPY packages ./packages
+
+RUN pnpm -r --filter "./packages/*" run build
+RUN pnpm --filter @smart-address/service-bun deploy --prod --legacy /app/deploy
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)